Multisig/UI/ClaimToken/EnterCustomAddressViewController.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  EnterCustomAddressViewController.swift
+//  Multisig
+//
+//  Created by Vitaly on 24.07.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+import Web3
+
+class EnterCustomAddressViewController: UIViewController {
```

Comment:
In general, I think we already have such a screen that asks for an address from the user. We should have used that one

Reply:
We do have different screens for address input but they differ in number of small details. I would not generalise this screen and keep it as simple as possible. This will allow us to be more flexible in the future, should any additional requirements / changes come that are related to this screen / flow.